django_admin_index/models.py

```diff
@@ -142,7 +150,7 @@ def __str__(self):
         return "{}.{}".format(self.app_label, capfirst(self.model))
 
 
-class AppGroup(OrderedModel):
+class AppGroup(OrderedModel, TranslationsMixin):
```

Comment:
For consistent MRO, it's convention to put mixins to the left and base classes to the right:

```suggestion
class AppGroup(TranslationsMixin, OrderedModel):
```